roles/mysql/tasks/main.yml

```
@@ -9,6 +9,15 @@
     - python3-mysqldb # required by Ansible mysql_db
     state: present
 
+- name: Enable mariadb error logging
+  replace:
+    path: /etc/mysql/mariadb.conf.d/50-server.cnf
+    # Uncomment matching lines
+    regexp: '^#\s?(\s*){{ item }}(.+)$'
+    replace: '\1{{ item }}\2'
+  loop:
+    - 'log_error'
```

This is a bit risky.  It would also match something like `log_error_destruction`. It also uses the value in the comment which could change if the default changes. We better leave the comment there and make sure that we have the right config option with the desired value. Example:
https://github.com/ceresfairfood/fairfood-install/blob/07eab77143a1588dbb96de1733d283ed8d97333e/roles/mail/tasks/main.yml#L12-L17
```suggestion
- name: Enable mariadb error logging
  lineinfile:
    dest: /etc/mysql/mariadb.conf.d/50-server.cnf
    regexp: "^log_error ?="
    line: "log_error = /var/log/mysql/error.log"
    state: present
```

---

Thanks, that's much better.
Any reason to leave the old comment there? I guess it's nice to keep a record of what the default was. I found that it was more intuitive to replace the old comment, so that it appears in the same part of the file (instead of appended at the bottom, under a different group).
What do you think about my amendment? (sorry I'll re-push with a separate commit, one sec)